edx_proctoring/templates/emails/proctoring_attempt_satisfactory_email.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<p>
+    {% blocktrans %}
+        Hi {username},
```

Comment:
Thanks, both of you. This is why I should have generated an email before submitting this for code review! I've fixed the code to use the correct syntax and pass the value through, and I've also extended the unit test to verify that the name is rendered correctly.